packages/xo-server-backup-reports/templates/mjml/partials/vmText.hbs

```diff
@@ -1,12 +1,14 @@
-
 {{#if vm}}
 <h3>{{vm.name_label}}</h3>
-<ul>
-  <li><span style="font-weight: bold">UUID</span>: {{vm.uuid}}</li>
 {{else}}
 <h3>VM not found</h3>
+{{/if}}
 <ul>
+{{#if vm}}
+  <li><span style="font-weight: bold">UUID</span>: {{vm.uuid}}</li>
+{{else}}
   <li><span style="font-weight: bold">UUID</span>: {{taskLog.data.id}}</li>
 {{/if}}
 {{>reportTemporalData end=taskLog.end start=taskLog.start}}
+{{>reportInfo task=taskLog}}
 {{>reportWarnings task=taskLog}}
```

Comment:
I would display warnings first because they are likely more important.

```suggestion
{{>reportWarnings task=taskLog}}
{{>reportInfo task=taskLog}}
```